pkg/chezmoi/archive.go

```diff
@@ -132,10 +134,44 @@ func isTarArchive(r io.Reader) bool {
 	return err == nil
 }
 
+func ImplicitDirHeader(dir string, fileHeader *tar.Header) *tar.Header {
+	return &tar.Header{
+		Typeflag: tar.TypeDir,
+		Name:     dir,
+		Mode:     0o755,
```

Comment:
This mode can be `0o777`. The user's umask will clear the appropriate group and world bits.